app/templates/ember_widgets/color_picker.hbs

```diff
@@ -6,24 +6,46 @@
     </div>
 
     <div class="col-md-3">
-      {{view Ember.Widgets.ColorPicker class="btn btn-default"}}
+      {{color-picker}}
       <button type="button" class="btn btn-default">Comparison</button>
     </div>
+    <div class="col-md-9">
+      <div class="highlight">
+        <pre class="prettyprint lang-html">
+          &#123;&#123;color-picker
+            class="btn btn-default"
+            dropdownClass="pull-left"
+          &#125;&#125;</pre>
+      </div>
+    </div>
 
     <div class="col-md-3">
-      {{view Ember.Widgets.ColorPicker class="btn btn-default" dropdownClass="pull-right"}}
+      {{color-picker class="btn btn-default" dropdownClass="pull-right"}}
```

Comment:
is there a reason why you let the `class` in that case and not in the previous example?